city_scrapers/spiders/chi_ssa_23.py

```diff
@@ -0,0 +1,143 @@
+import logging
+import re
+import unicodedata
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa23Spider(CityScrapersSpider):
+    name = "chi_ssa_23"
+    agency = "Chicago Special Service Area #23 Clark Street"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.lincolnparkchamber.com/clark-street-ssa-administration/"]
+    location = {
+        "name": "Lincoln Park Chamber of Commerce",
+        "address": "2468 N. Lincoln Chicago, IL 60614"
+    }
+    # Each meeting takes place on Wednesday 4 PM
+    meeting_day = 'Wednesday'
+    time = '4 pm'
+
+    def parse(self, response):
+        # Due to the current lack of documents for the meetings of 2020
+        # no assumption is made regarding the expected HTML page format
+        # once these are uploaded.
+        h4s = response.xpath('//h4')
+        meeting_years = list()
+
+        for entry in h4s.xpath('./text()').getall():
+            # Do not allow duplicates
+            if entry[0:4] not in meeting_years:
+                meeting_years.append(entry[0:4])
+
+        # General meeting description is mentioned just after the H4 for the current year
+        general_desc = h4s.xpath('following-sibling::p[1]//em//text()').extract_first()
+
+        # List containing all meeting dictionaries
+        meetings = list()
+
+        link_collection = []
+        for h4 in h4s[1:]:
+            # Get the links for each meeting
+            link_collection.append(h4.xpath('following-sibling::p[1]//a'))
+
+        last_year = datetime.today().replace(year=datetime.today().year - 1)
+
+        for year_counter, year in enumerate(meeting_years):
+
+            if year_counter == 0:
+                for item in h4s.xpath('following-sibling::ol[1]//li//text()').extract():
+                    start, end = self._parse_start_end(item, meeting_years[0])
+
+                    meetings.append({
+                        'start': start,
+                        'end': end,
+                        # Currently no meetings from 2020 have links
+                        'links': None
```

Comment:
This should return an empty list if no links are available, not `None`

Reply:
Makes sense, thanks. I changed this part.